lms/services/oauth_http.py

```diff
@@ -104,7 +104,7 @@ def refresh_access_token(self, token_url, redirect_uri, auth):
         except ExternalRequestError as err:
             try:
                 error_dict = _OAuthAccessTokenErrorResponseSchema(err.response).parse()
-            except ValidationError:
+            except ExternalRequestError:
```

Comment:
`_OAuthAccessTokenErrorResponseSchema` (which is private to this file) is a `RequestsResponseSchema` subclass so it now raises `ExternalRequestError` rather than `ValidationError`